custom_components/deebot/const.py

```diff
@@ -44,4 +34,17 @@
     "STATE_DOCKED": STATE_DOCKED,
     "STATE_ERROR": STATE_ERROR,
     "STATE_PAUSED": STATE_PAUSED,
-}
\ No newline at end of file
+}
+
+CONF_BUMPER = "Bumper"
+CONF_MODE_BUMPER = CONF_BUMPER
+CONF_MODE_CLOUD = "Cloud (recommended)"
+
+# Bumper has no auth and serves the urls for all countries/continents
+BUMPER_CONFIGURATION = {
```

Comment:
Here is the dummy bumper configuration, which will set bumper for both username and password.

The real different between the two modes is, that when selecting bumper the verify_ssl will be set to False. Otherwise it will be true.